index.html

```diff
@@ -299,49 +311,61 @@ <h3 id="advertising-payload">Advertising payload</h3>
+          the <strong>BTHome Device Information</strong> and one or
+          multiple <strong>Measurements</strong>. <br /><br />In the
+          example we have the following BTHome data:<br />
+          <code>0xFCD2 420100 2302C409 0303BF13</code>
+        </p>
+        <strong>UUID (<code>0xFCD2</code>)</strong><br />
+        <p>
+          This UUID should be used by receivers to recognize BTHome
+          messages.<br /><br />
+          Allterco Robotics, the manufacturer of Shelly devices, has sponsored the UUID
+          <code>0xFCD2</code> for BTHome and it's free to use for everyone, as
+          long as you use it with the BTHome format.
+        </p>
+        <strong>BTHome Device Information (<code>0x42 0200</code>)</strong><br />
+        <p>
+          The first byte after the UUID <code>0x42</code> is the BTHome device info byte,
+          which has several bits indicating the capabilities of the device.
+        </p>
+        <ul>
+          <li>bit 0: “Encryption flag”</li>
+          <li>bit 1: “Predefined device information flag”</li>
+          <li>bit 2-4: “Reserved for future use”</li>
+          <li>bit 5-7: “BTHome Version”</li>
+        </ul>
         <p>
-          The BTHome data can contain multiple measurements. We continue with
-          the example from above.
+          In the example, we have <code>0x42</code>.
+          In bits, this is <code>01000010</code>.
+          Note. bit 0 is the most right number, bit 7 is the most left bit<br />
         </p>
-        BTHome data =
-        <code>0x1C18 2302C409 0303BF13</code>
         <ul>
-          <li>
-            <code>0x1C18</code>
-            = The first two byte are the UUID16. Use <code>0x1C18</code> for
-            non-encrypted messages and <code>0x1E18</code> for encrypted data.
-            This data should be used by receivers to recognize BTHome
-            messages.<br /><br />More information about encryption can be found
-            <a href="#encryption">further down this page</a>.
-          </li>
-          <li>
-            <code>0x23 02 C409</code>
-            = Temperature packet
-          </li>
-          <li>
-            <code>0x03 03 BF13</code>
-            = Humidity packet
-          </li>
+          <li>bit 0: <code>0</code> = False: No encryption</li>
+          <li>bit 1: <code>1</code> = True: Use predefined device information</li>
+          <li>bit 5-7: <code>010</code> = 2: BTHome Version 2. <br />
+            Other versions numbers are currently not allowed</li>
         </ul>
         <p>
-          Lets explain how the last two data packets work. The temperature
-          packet is used as example. The first byte
+          Because we have set the predefined device information flag to True,
+          we have to add two extra bytes that correspond to a device in the
+          <a href="https://github.com/Bluetooth-Devices/bthome-ble/tree/main/src/bthome_ble/">devices.py</a> file. In the
```

Comment:
This doesn't link to `devices.py`. Also, this doesn't seem like it will scale? I don't think that we should carve out special identifiers for early adopters.

Reply:
We should enumerate devices in a language-agnostic data format -- json, csv or (god forbid) xml. BTHome tries to be a standard, after all, implementations should follow.

Reply:
As discussed, I removed the predefined device information flag and bytes, as we will solve that later by reading GATT characteristics with the device info. Added a device_info_available flag for that, for a future release.